### ISSUE TITLE
Only one of my models isn't showing up, in production

### ISSUE THREAD

**Issue by jc00ke** [railsadminteam/rails_admin#1195]
I have 7 models, and all 7 show up in development. When I push to production, only 6 show up. The one that doesn't, let's call it `Foo`, is of course available in the console, and is even configured in `config/initializers/rails_admin.rb`

Kinda stumped. Any ideas on what info I can provide? I'm on EY using puma, 1.9.3-p125 and pg. I just upgraded to latest rails_admin `0.0.4` today and rails `3.2.5`. Everything else works!

**Comment by DeweyBanks** [comment]
This is for anyone else who runs across this in the future.
My model wasn't showing up in the Rails Admin because I was using
config.included_models = ["ModelOne", "ModelTwo" "ModelThree"]
if you notice there is no comma between ModelTwo and ModelThree. 
Point is, check for dumb little mistakes

**Comment by aop4** [comment]
For me, this was caused by a silly issue. I had 0 models of a certain kind in production. When I went to create a new instance (at <base_url>/admin/model_name/new), a blank page showed up. It took me a while to think of looking through my production logs (try it). When I did, I saw this snippet `relation "active_storage_attachments" does not exist`, and I knew what was up. I just forgot to run migrations.
